### PR TITLE
Update npm dependencies

### PR DESCRIPTION
**Pull request by christianbundy**
Problem: Our dependencies are falling behind and it'd be nice to have
them up-to-date. The only big update it that Common-Good upgraded to the
latest version of Prettier.

Solution: Update them!

### CODE REVIEW

**Comment by thorn0** [comment]
It's `prettier --write`, not `--fix`.

**Comment by thorn0** [comment]
`&& git diff` → `|| git diff`

**Comment by christianbundy** [comment]
Oh, thanks. I assumed that `prettier --write .` would also exit 0, but maybe it has errors too. I'll change to your suggestion once this commit fails like the others. Sorry, my brain's a bit fried.

**Comment by thorn0** [comment]
Um, you're actually right. `--write` should return 0.

**Comment by thorn0** [comment]
Try the `file` command, from https://stackoverflow.com/a/3570574/76173 :
```console
$ file testfile1.txt
testfile.txt: ASCII text

$ file testfile2.txt
testfile2.txt: ASCII text, with CRLF line terminators
```

**Comment by christianbundy** [comment]
Resolved the above problems in https://github.com/prettier/prettier/issues/7853. :tada: Thanks for your help!

File: package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
   },
   "dependencies": {
     "@fraction/base16-css": "^1.1.0",
-    "@fraction/flotilla": "^4.0.1",
+    "@fraction/flotilla": "^4.0.0",
```
Comment (jedahan) [inline package.json]: flotilla downgrade?

Comment (christianbundy) [reply package.json]: Oops, unintentional. I'll fix that now.

**Comment by christianbundy** [comment]
@jedahan fixed! please merge when you're happy with this!

**Comment by jedahan** [comment]
once the conflict is fixed i'll be happy to
though i cannot test it, the updates look fine

**Comment by christianbundy** [comment]
Done! Ready to go now.

**Comment by christianbundy** [comment]
Thanks for the merge!

**Pull request merged**